### ISSUE TITLE
[ActionButton] toggling does not have sufficient colour contrast between selected/active and default state

### ISSUE THREAD

**Issue by najikahalsema** [adobe/spectrum-css#1165]
## Description!
The difference between an Action Button in its default state (ie not being interacted with) and its pressed state does not fall within the required 3:1 luminosity contrast ratio requirements. This issue was brought up at the SWC level, but we don't necessarily have the authority to make this change to colour.  

The colour values are
Unselected: #FAFAFA in Hex, RGB(234, 234, 234)
Selected: #EAEAEA in Hex, RGB(250, 250, 250)

The contrast ratio between these two values is 1.15 : 1 according to [WebAIM's contrast checker](https://webaim.org/resources/contrastchecker/?fcolor=FAFAFA&bcolor=EAEAEA).

## Steps to reproduce

Please view the action button here: https://opensource.adobe.com/spectrum-web-components/components/action-button/examples

You can get the colour of the unselected button, the selected button, and compare them in the [contrast checker](https://webaim.org/resources/contrastchecker/). This would be problematic for some of our users who require a stronger visual difference between the two button states.

## Expected behavior

An accessible version of these colours for our users.

## Screenshots

![image](https://user-images.githubusercontent.com/44980010/114630434-6ed9c300-9c6f-11eb-8062-16c029b4295a.png)

## Environment
 - **Spectrum CSS version:** 3.0.1
 - **Browser(s) and OS(s):** Chrome 89.0.4389.90 on MacOS Big Sur 11.0.1

## Additional context

Issues previously reported in SWC:
https://github.com/adobe/spectrum-css/issues/1106

**Comment by castastrophe** [comment]
These issues appear to be resolved now: https://opensource.adobe.com/spectrum-css/?path=/story/components-action-button--default&globals=testingPreview:!true
